src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -2038,6 +2038,19 @@
   (jdbc/do-commands
    "ALTER TABLE certnames ADD COLUMN catalog_inputs_hash bytea"))
 
+(defn add-workspaces-tables
+  []
+  (jdbc/do-commands
+   ["CREATE TABLE workspaces ("
+    "  workspace_id UUID PRIMARY KEY,"
+    "  updated TIMESTAMP WITH TIME ZONE NOT NULL)"]
+
+   ["CREATE TABLE workspace_membership ("
```

Comment:
```suggestion
   ["CREATE TABLE workspace_memberships ("
```

I think this table name should be plural as well. Doesn't really matter one way or the other but most of our table names are plural